.github/workflows/genesis-testnet.yml

```diff
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     defaults:
       run:
-        working-directory: ./solidity
+        working-directory: ./solidity-v1
     steps:
       - uses: actions/checkout@v2
 
```

Comment:
`cache-dependency-path` below also has to be updated

Reply:
bf8f2de